tardis/visualization/tools/tests/test_lineid_plotter.py

```diff
@@ -3,14 +3,8 @@
 from matplotlib.testing.compare import compare_images
 
 # check if lineid_plot is installed
-try:
-    import lineid_plot
-
-    lineid_installed = True
```

Comment:
The removal of this variable is causing errors in the tests, looks like it is still used somewhere else.